src/stylesheets/screens/question.scss

```diff
@@ -0,0 +1,69 @@
+.questions {
+  width: 100vw;
+  height: 100vh;
+
+  &__header {
+    @include header-container();
+    height: rem(60px);
+
+    padding: rem(16px 16px 0);
+  }
+
+  &__header-right {
```

Comment:
Avoid using words which describe the placement of UI elements (`-right`) on the screen as the design can change and be different on different screen sizes. Instead, prefer using behaviour description e.g. `header-control`, `header-action`...